### PR TITLE
Update CLI reference docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2/issues/2250 and https://github.com/blackbaud/skyux2/issues/2195. And https://github.com/blackbaud/skyux2/issues/2251.

### CODE REVIEW

File: src/app/learn/reference/cli-commands/install/index.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<stache
+  pageTitle="install">
+
+  <p>
+    <stache-code>skyux install</stache-code> deletes the <stache-code>node_modules</stache-code> folder and runs <stache-code>npm install</stache-code>. This clears the existing dependencies in the <stache-code>node_modules</stache-code> folder and ensures that all SKY UX dependencies are consistent and up-to-date.
```
Comment (Blackbaud-BobbyEarl) [inline src/app/learn/reference/cli-commands/install/index.html]: I'm worried the "up-to-date" may cause ambiguity.  Perhaps we could add "up-to-date based on your `package.json`" or remove it all together.  Just don't want anyone to think that this command updates the versions, it simply installs the versions that match whatever's specified in their `package.json`.

Comment (blackbaud-johnly) [reply src/app/learn/reference/cli-commands/install/index.html]: Not sure when you are going to get another chance to look at this, so I'm going to go with the latter option.

File: src/app/learn/reference/cli-commands/install/index.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<stache
+  pageTitle="install">
+
+  <p>
+    <stache-code>skyux install</stache-code> deletes the <stache-code>node_modules</stache-code> folder and runs <stache-code>npm install</stache-code>. This clears the existing dependencies in the <stache-code>node_modules</stache-code> folder and ensures that all SKY UX dependencies are consistent and up to date based on your <stache-code>package.json</stache-code> file.
```
Comment (Blackbaud-SteveBrush) [inline src/app/learn/reference/cli-commands/install/index.html]: Maybe something like, “...SKY UX dependencies are consistent with your `package.json` file.“? The feature essentially deletes the node_modules folder and reinstalls all dependencies as they are defined in package.json.

Comment (blackbaud-johnly) [reply src/app/learn/reference/cli-commands/install/index.html]: I'll make the change.

**Pull request merged**